portal/server/controllers/dashboard/facilities/template-filters.ff-test.js

```diff
@@ -0,0 +1,36 @@
+import { FACILITY_STATUS } from '@ukef/dtfs2-common';
+import { stageFilters } from './template-filters';
+import { FIELD_NAMES } from '../../../constants';
+import { DASHBOARD_FILTERS } from '../../../content-strings';
+import { generateFiltersArray } from '../filters/generate-template-filters';
+
+describe('controllers/dashboard/facilities - template-filters', () => {
+  describe('stageFilters - FF_TFM_FACILITY_END_DATE_ENABLED enabled', () => {
```

Comment:
Can this be "facility stage filters"  instead of "stageFilters", threw me a little bit